.github/workflows/pyink.yml

```diff
@@ -28,6 +28,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+        with:
```

Comment:
Why adding this? The default is checking the "merged version", which would be better to CI/CD.